database/seeders/DatabaseSeeder.php

```diff
@@ -12,13 +15,21 @@ class DatabaseSeeder extends Seeder
      *
      * @return void
      */
-    public function run()
-    {
-        // \App\Models\User::factory(10)->create();
+    public function run(){
```

Comment:
```suggestion
    public function run(){
       $this->call([CategorySeeder::class,]);
```

Reply:
Makes sense, I can update this Sunday evening

Reply:
this is correct